src/Functions/DateTimeTransforms.h

```diff
@@ -1446,26 +1479,34 @@ struct Transformer
     }
 };
 
-
 template <typename FromDataType, typename ToDataType, typename Transform, bool is_extended_result = false>
 struct DateTimeTransformImpl
 {
+    template <typename Additions = void *>
```

Comment:
`Additions` is a quite generic name, but I can't really think of a better one.

EDIT: Maybe `ErrorHandlingStrategy` or just `ErrorHandling`?